pkg/appgw/appgw_test.go

```diff
@@ -306,8 +306,8 @@ var _ = Describe("Tests `appgw.ConfigBuilder`", func() {
 		err = configBuilder.BackendHTTPSettingsCollection(ingressList, serviceList)
 		Expect(err).Should(BeNil(), "Error in generating the HTTP Settings: %v", err)
 
-		// Retrieve the implementation of the `ConfigBuilder` interface.
-		appGW, _ := configBuilder.Build(ingressList, serviceList)
+		// Get pointer the modified ApplicationGatewayPropertiesFormat
```

Comment:
How about `// Get __a__ pointer __to__ the modified ...` for this and the other comments?